projects/components/table/src/subcomponents/table-actions.component.html

```diff
@@ -21,15 +25,28 @@
           [routerLink]="action.routerLink(items).path"
           [queryParams]="action.routerLink(items).queryParams"
         >
-          <ng-template [ngTemplateOutlet]="actionIcon" [ngTemplateOutletContext]="{ $implicit: action }"></ng-template>
+          <ng-template
+            *ngIf="showIcon || action.icon"
```

Comment:
What is the || action.icon part for? It didn't break the template before if the icon was null. And this change does not correspond with the title of the pull request.
the showIcon check should do that already

Reply:
you are right. i change it.